### PR TITLE
[Performance] Lazy Load all iframes

### LINKED ISSUE #2623: Lazy load all iframes
**Current Behavior**
We use several `<iframes>` across [Resources](https://layer5.io/resources), [Blogs](https://layer5.io/blog/meshery/an-introduction-to-meshery-webinar-on-demand),  [Events](https://layer5.io/community/events/kubecon-china-2021).

**Desired Situation**
Use the lazy load attribute for each of these.

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by sidd-oo**
Signed-off-by: sidd-oo <sahoosiddharth3@gmail.com>

**Description**
In this PR I have used lazy loading attribute for every iframe present, there are several `<iframes>` across the layer5 website like in the [Resources](https://layer5.io/resources), [Blogs](https://layer5.io/blog/meshery/an-introduction-to-meshery-webinar-on-demand), [Events](https://layer5.io/community/events/kubecon-china-2021).

Lazy loading is a strategy to identify resources as non-blocking (non-critical) and load these only when needed. It's a way to shorten the length of the [critical rendering path](https://developer.mozilla.org/en-US/docs/Web/Performance/Critical_rendering_path), which translates into reduced page load times.

This PR fixes #2623 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by sidd-oo** [comment]
@debo19 Kindly review the change. Thank you so much.

**Comment by l5io** [comment]
🚀 Preview for commit 43545ac77ba64dcb7fa055afad1d92e17ac9c73b at: https://621cc19483b7b23c65e5d7d9--layer5ng.netlify.app

File: src/collections/events/istiocon2020/index.mdx [hunk 1]
```diff
@@ -42,6 +42,6 @@ See link for self-paced study.
 <br />
 <h4 style="margin-bottom:1.25rem;">Workshop Recording</h4>
 
-<iframe width="50%" src="https://www.youtube.com/embed/T1Kwh3SRAXQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" style="min-height: 315px; min-width: 280px;margin:auto;text-align:center;" allowfullscreen></iframe>
+<iframe width="50%" src="https://www.youtube.com/embed/T1Kwh3SRAXQ" loading="lazy" loading="lazy" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" style="min-height: 315px; min-width: 280px;margin:auto;text-align:center;" allowfullscreen></iframe>
```
Comment (Abhijay007) [inline src/collections/events/istiocon2020/index.mdx:45]: @sidd-oo, remove duplicate attribute.

Comment (sidd-oo) [reply src/collections/events/istiocon2020/index.mdx:45]: Thanks, @Abhijay007 I am on it.

**Comment by l5io** [comment]
🚀 Preview for commit 58e8aa613fc74a426ac60856ac658d7670427f50 at: https://621cc9c183b7b24220e5d7d4--layer5ng.netlify.app

**Comment by sidd-oo** [comment]
Kindly take a look now @Abhijay007 @debo19

**Review by YashKamboj — APPROVED** [review]

**Review by Abhijay007 — APPROVED** [review]
LGTM !!

**Review by debo19 — APPROVED** [review]
LGTM 🎉 Thanks @sidd-oo

**Pull request merged**